.gitmodules

```diff
@@ -1428,3 +1428,6 @@
 [submodule "vendor/grammars/zephir-sublime"]
 	path = vendor/grammars/zephir-sublime
 	url = https://github.com/phalcon/zephir-sublime
+[submodule "vendor/grammars/Yooz.tmbundle"]
+	path = vendor/grammars/Yooz.tmbundle
+	url = https://github.com/ai-yooz/simple-yooz-code
```

Comment:
This is not a TextMate compatible grammar. The fact this is here indicates you have NOT followed the directions in CONTRIBUTING.md